### ISSUE TITLE
Menu options not appearing next to menu on smaller screen

### ISSUE THREAD

**Issue by GammaVel** [constellation-app/constellation#1896]
### Prerequisites

- [X] Put an X between the brackets on this line if you have done all of
    the following:

    -   Running the latest version of Constellation

    -   Attached the ***Support Package*** via `Help` &gt;
        `Support Package`

    -   Checked the FAQs:
        https://github.com/constellation-app/constellation/wiki/FAQ

    -   Checked that your issue isn’t already filed:
        https://github.com/constellation-app/constellation/issues

    -   Checked that there is not already a module that provides the
        described functionality:
        https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

I have noticed for a while that when I run Constellation from my laptop which is set to 1920x1200 (which is the recommended setting for my laptop) that menu items and hover text can appear at a distance from the original item. This is ok if you are fairly familiar with the application but a bit confusing if you are new.  I have attached some examples of this happening below.
I also have an issue with the text appearing very small, when using the default settings which was raised in #1062 (it looks like nothing can be done with the version of netbeans for this). 

### Steps to Reproduce

Data Access View > seeking to Adjust the Remove By for the Remove note utility:
![image](https://github.com/constellation-app/constellation/assets/83679004/dfa21de2-71a2-449d-aada-7aa341871553)

Here I am trying to change the date time formatter when importing a CSV to construct a graph:
![image](https://github.com/constellation-app/constellation/assets/83679004/7534e48c-98d2-4548-9687-88309e453b76)

Here I am merging nodes:
![image](https://github.com/constellation-app/constellation/assets/83679004/b670e796-4ce4-4ca8-a168-3bffe8ae44fd)

Here the hover text is far away from the actual graph that it belongs to on the welcome page:
![image](https://github.com/constellation-app/constellation/assets/83679004/701696fe-945d-4e17-ac2b-401b923ccfc1)

### Behaviour

* Try to include measurable metrics that can be aimed for (if applicable to the issue),
e.g. if it's a time issue then "should take 5 seconds or less to finish"
or if it's a workflow issue "should be accomplished in 2 or less clicks".

-->

**Expected behaviour:** Any menu items appear next to where the functionality is.

**Actual behaviour:**The menu items appear further away and not related to the menu

**Reproduces how often:**100% - this is not an issue on my wide screen monitor. 

### Additional Information

Small font was previously raised

**Comment by serpens24** [comment]
I'm wondering if the issue here is windows scaling. Ive seen similar issues in .NET applications where things line up nicely at 100% scale, but changing to other values resutls in all sort of issues.

I tried the CSV import example above at 100% scal and it looked fine, but at 150% scale the menu was displayed roughly 2/3 as low as I expected it.

**Comment by GammaVel** [comment]
Good point @serpens24, I thought I was running at 100% but when I checked Windows was set to 125% which is the recommended setting. When I changed it to 100% it immediately fixed my issue of the menus not appearing close enough to the menu that they relate to.
